src/pages/Payment/Payment.jsx

```diff
@@ -31,24 +44,44 @@ const Payment = () => {
       setIsCheckedTerms(false);
       setIsDisabledPayment(true);
     } else {
-      if (paymentPrice === 0) {
+      if (paymentPrice === 0 && deliveryValueCheck) {
         setIsCheckedTerms(true);
         setIsDisabledPayment(false);
       }
     }
   };
-  const showModal = () => {
-    setModalOpen(true);
+
+  const showOrderComplete = () => {
+    fetch(`${PAYMENT_API}`, {
```

Comment:
```suggestion
    fetch(PAYMENT_API, {
```
변수에 주소값이 할당되어있어서, 템플릿리터럴 사용하지않아도 스트링으로 참조 됩니다